### PR TITLE
zh-translation: content\zh\docs\tasks\traffic-management\ingress\gateway-api\index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+weight: 50
+aliases:
+    - /docs/tasks/traffic-management/ingress/service-apis/
+    - /latest/docs/tasks/traffic-management/ingress/service-apis/
+keywords: [traffic-management,ingress]
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+本任务描述如何配置 Istio ，以使用 Kubernetes Gateway API 在 service mesh 集群外部暴露服务。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:13]:
```suggestion
本任务描述如何配置 Istio ，以使用 Kubernetes Gateway API 在 Service Mesh 集群外部暴露服务。
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+## 设置 {#setup}
+
+1. 在大多数 Kubernetes 集群中，默认情况下不会安装 Gateway API。如果 Gateway API CRD 不存在，请安装：
+
+    {{< text bash >}}
+    $ kubectl get crd gateways.gateway.networking.k8s.io || { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0" | kubectl apply -f -; }
+    {{< /text >}}
+
+## 与 Istio API 的区别{#Differences-from-Istio-APIs }
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:31]:
```suggestion
## 与 Istio API 的区别{#differences-from-Istio-APIs }
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+Gateway API 与 Istio API (如 Gateway 和 VirtualService )有很多相似之处。
+主资源使用相同的  `Gateway` 名称，并且这些资源服务于相类似的目标。
+
+新的 Gateway API 致力于从 Kubernetes 的各种 Ingress 实现(包括 Istio )中吸取经验，以构建标准化的，独立于供应商的 API。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:36]:
```suggestion
新的 Gateway API 致力于从 Kubernetes 的各种 Ingress 实现（包括 Istio）中吸取经验，以构建标准化的，独立于供应商的 API。
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    
+1.  访问其他没有被显式暴露的 URL 时，将看到 HTTP 404 错误：
+
+    {{< text bash >}}
+    $ curl -s -I -HHost:httpbin.example.com "http://$INGRESS_HOST/headers"
+    HTTP/1.1 404 Not Found
+    ...
+    {{< /text >}}
+
+## 部署方法{#Deployment-methods}
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:134]:
```suggestion
## 部署方法{#deployment-methods}
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+在上面的示例中，在配置网关之前，你不需要安装 ingress 网关 `Deployment` 。因为在默认配置中会根据 `Gateway` 配置自动分发网关`Deployment` 和 `Service` ，但是对于高级别的用例，仍然允许手动部署。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:136]:
```suggestion
在上面的示例中，在配置网关之前，您不需要安装 ingress 网关 `Deployment` 。因为在默认配置中会根据 `Gateway` 配置自动分发网关`Deployment` 和 `Service` ，但是对于高级别的用例，仍然允许手动部署。
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+
+### 自动部署{#Automated-Deployment}
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:138]:
```suggestion
### 自动部署{#automated-Deployment}
```

**Review by kebe7jun — COMMENTED** [review]
ID 除了专有名词小写。

使用`您`代替`你`。

**Comment by Xunzhuo** [comment]
@yanrongshi Thanks for your contributions 👍. Please fix some lint errors first.

**Comment by ericvn** [comment]
Lint errors can be found by clinking on the `Details` after the test and looking at the logs.

File: content/zh/news/releases/1.12x/announcing-1.12/_index.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (kebe7jun) [inline content/zh/news/releases/1.12x/announcing-1.12/_index.md:1]: Don't add this file in this PR, create a new PR do this.

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+        ...
+        {{< /text >}}
+
+请注意:仅能指定一个地址。
+
+* (高级用法)生成的 Pod 配置可以通过[自定义注入模板](/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental)进行配置。
+
+### 手动部署{#manual-deployment}
+
+如果您不希望使用自动部署，可以进行[手动配置](/docs/setup/additional-setup/gateway/) `Deployment` 和 `Service`。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:171]:
```suggestion
如果您不希望使用自动部署，可以进行[手动配置](/zh/docs/setup/additional-setup/gateway/) `Deployment` 和 `Service`。
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+      addresses:
+    
+      - value: 192.0.2.0
+        type: IPAddress
+        ...
+        {{< /text >}}
+
+请注意:仅能指定一个地址。
+
+* (高级用法)生成的 Pod 配置可以通过[自定义注入模板](/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental)进行配置。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:167]:
```suggestion
* (高级用法)生成的 Pod 配置可以通过[自定义注入模板](/zh/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental)进行配置。
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+      name: gateway
+    spec:
+      addresses:
+    
+      - value: 192.0.2.0
+        type: IPAddress
+        ...
+        {{< /text >}}
+
+请注意:仅能指定一个地址。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:165]:
```suggestion
请注意：仅能指定一个地址。
```

**Review by kebe7jun — COMMENTED** [review]
注意下细节。

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 11]
```diff
@@ -0,0 +1,217 @@
...
+weight: 50
+aliases:
+    - /zh/docs/tasks/traffic-management/ingress/service-apis/
+    - /latest/zh/docs/tasks/traffic-management/ingress/service-apis/
+keywords: [traffic-management,ingress]
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+本任务描述如何配置 Istio ，以使用 Kubernetes Gateway API 在 Service Mesh 集群外部暴露服务。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:13]:
```suggestion
本任务描述如何配置 Istio，以使用 Kubernetes Gateway API 在 Service Mesh 集群外部暴露服务。
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 12]
```diff
@@ -0,0 +1,217 @@
...
+    $ curl -s -I -HHost:httpbin.example.com "http://$INGRESS_HOST/get"
+    HTTP/1.1 200 OK
+    server: istio-envoy
+    ...
+    {{< /text >}}
+
+    请注意，使用 `-H` 标志可以将 *Host* HTTP 标头设置为
+    "httpbin.example.com"。这一步是必需的，因为 `HTTPRoute` 已配置为处理"httpbin.example.com"的请求，
+    但是在测试环境中，该主机没有 DNS 绑定，只是将请求发送到入口 IP。
+
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:127]:
```suggestion

```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 13]
```diff
@@ -0,0 +1,216 @@
...
+aliases:
+    - /zh/docs/tasks/traffic-management/ingress/service-apis/
+    - /latest/zh/docs/tasks/traffic-management/ingress/service-apis/
+keywords: [traffic-management,ingress]
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+本任务描述如何配置 Istio，以使用 Kubernetes Gateway API 在 Service Mesh 集群外部暴露服务。
+这些 API 是 Kubernetes [Service](https://kubernetes.io/zh/docs/concepts/services-networking/service/)
```
Comment (yanrongshi) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:14]:
```suggestion
这些 API 是 Kubernetes [Service](https://kubernetes.io/zh/docs/concepts/services-networking/service/)
```

**Review by kebe7jun — APPROVED** [review]

**Comment by kebe7jun** [comment]
thanks！

**Pull request merged**